src/claude-session-utility.ts

```
@@ -1411,12 +1416,30 @@ export const launchNewClaudeSession = (
-  runCommandInTerminal(`cd "${projectPath}" && claude`, 'claude', projectPath, terminalApp, terminalMode);
+  // No accountLabel: bare `claude` → the accounts.sh dispatcher → the user's
+  // global default (2e). With an explicit account (2c-lite picker): bypass the
+  // dispatcher via `command claude` + explicit env, like buildResumeCommand.
+  let claudeCmd = 'claude';
+  if (accountLabel) {
+    const account = getAccountByLabel(accountLabel);
+    claudeCmd = account?.configDirEnv
+      ? `CLAUDE_CONFIG_DIR='${account.configDirEnv.replace(/'/g, "'\\''")}' command claude`
+      : 'command claude';
+  }
+  runCommandInTerminal(`cd "${projectPath}" && ${claudeCmd}`, 'claude', projectPath, terminalApp, terminalMode);
```

ESLint

[error] Replace ``cd·"${projectPath}"·&&·${claudeCmd}`,·'claude',·projectPath,·terminalApp,·terminalMode` with `⏎····`cd·"${projectPath}"·&&·${claudeCmd}`,⏎····'claude',⏎····projectPath,⏎····terminalApp,⏎····terminalMode,⏎··`

(prettier/prettier)